src/store/index.js

```diff
@@ -6,33 +6,55 @@ import router from '../router/index.js';
 
 Vue.use(Vuex);
 
+const github = {
+  url: 'http://localhost:8000/api/v1/auth/login/',
+  clientId: 'b1566423c56583f16eb0',
+  redirectUri: 'http://localhost:8080/auth/callback'
+};
+
 const API_URL = 'http://localhost:8000/api/v1';
 
 export default new Vuex.Store({
   state: {
     currentUser: {},
-    lastURLRequested: null
+    lastURLRequested: null,
+    isAuthenticated: false
```

Comment:
```suggestion
    lastURLRequested: null
```
doesn't seem to be used anywhere. easy to add back in if needed, but will probably just use `currentUser` to check, unless I'm misunderstanding some of the auth/local storage set up where they could have a currentUser, but not be auth'd